integration/spring-dsl-integration/docker/docker-compose.yml

```
@@ -0,0 +1,43 @@
+name: spring-dsl-integration
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)